### PR TITLE
Update Graphics and Tutorials Questionnaire

### PR DESCRIPTION
**Pull request by ms10398**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -472,74 +472,100 @@ let questionnaire = [
         'Select your option',
         'More than 3 distinctive, relatively complex designs were provided.',
         'Between 2 and 3; the designs were complex and took a fair amount of work.',
-        'Only one distinctive design was provided or the provided designs are too similar.',
+        'Only one distinctive design was provided among the provided designs.',
+        'Only Single Design was provided.',
```
Comment (espoem) [inline routes/index.js]: Change to "Only single design was provided."

File: views/index.ejs [hunk 2]
```diff
@@ -980,43 +984,22 @@
...
-                  						<select id="6">
-                  					<option value="" disabled selected>Select your option</option>
-                  					<option value="0">Yes, and the contributor provided clear evidence of it in their post.</option>
-                  					<option value="-4">It was not used, but the project owner liked it and considered using it.</option>
-                  					<option value="-5">It was not used, but is of very high quality.</option>
-                  					<option value="-8">It was not used as it provides no value to the project.</option>
+                  					<option value="7">More than 3 distinctive, relatively complex designs were provided.</option>
+                  					<option value="0">Between 2 and 3; the designs were complex and took a fair amount of work.</option>
+														<option value="-3">Only one distinctive design was provided among the provided designs.</option>
+														<option value="-6">Only Single Design was provided.</option>
```
Comment (espoem) [inline views/index.ejs]: Change to "Only single design was provided."

File: views/index.ejs [hunk 3]
```diff
@@ -1025,32 +1008,78 @@
                   						<select id="6">
                   					<option value="" disabled selected>Select your option</option>
                   					<option value="0">Excellent, no mistakes within the design.</option>
-                  					<option value="-16">Very good with minor mistakes that are visible in the provided files.</option>
-                  					<option value="-26">Good but the design has some more serious flaws that need to be corrected.</option>
-                  					<option value="-37">Bad. The design does not follow graphics standard and is barely usable.</option>
+                  					<option value="-3">Very good, open for improvements.</option>
+                  					<option value="-26">Good but the design has some serious flaws that needs to be corrected.</option>
```
Comment (espoem) [inline views/index.ejs]: Can you change "flaws that needs to" to "flaws that need to"? Missed typo in the doc.

File: routes/index.js [hunk 4]
```diff
@@ -472,74 +472,100 @@ let questionnaire = [
...
+      question: 'Technical quality of graphics was:',
       answers: [
         'Select your option',
-        'Yes.',
-        'Some third party assets were used but all of them are usable for the contribution actual purpose. The assets were accordingly edited and not simply downloaded and used.',
-        'Other assets were provided by the project owner.',
-        'Most assets used were third party assets and contributor did not make additional effort to incorporate them in the contribution.'
+        'Excellent, no mistakes within the design.',
+        'Very good, open for improvements.',
+        'Good but the design has some serious flaws that needs to be corrected.',
```
Comment (espoem) [inline routes/index.js]: Can you change "flaws that needs to" to "flaws that need to"? Missed typo in the doc.

File: routes/index.js [hunk 5]
```diff
@@ -472,74 +472,100 @@ let questionnaire = [
...
+        'Average. The post contains mistakes that should be corrected.',
+        'Low Quality. The post contains many mistakes in writing.',
       ]
     },
     {
       question:
-        'Does the contribution include files for immediate use in all requested formats?',
+        'What is the visibility impact on OS community and the popularity of the project?',
       answers: [
         'Select your option',
```
Comment (espoem) [inline routes/index.js]: Can you change "more than" to "greater than" in all the answers?

File: views/index.ejs [hunk 6]
```diff
@@ -1025,32 +1008,78 @@
...
+                  					<option value="-10">Average. The post contains mistakes that should be corrected.</option>
+                  					<option value="-28">Low Quality. The post contains many mistakes in writing.</option>
                   					</select>
                   					</div>
                   
                   						<div class="question-block">
-                  						<h6>How would you rate the overall value of this contribution on the open source community and ecosystem?</h6>
+                  						<h6>What is the visibility impact on OS community and the popularity of the project?</h6>
                   						<select id="7">
                   					<option value="" disabled selected>Select your option</option>
```
Comment (espoem) [inline views/index.ejs]: Can you change "more than" to "greater than" in all options here?

File: routes/index.js [hunk 7]
```diff
@@ -638,7 +664,8 @@ let questionnaire = [
         '4-5 substantial concepts covered in the tutorial',
         '2-3 substantial concepts covered in the tutorial',
         '1 substantial concept covered in the tutorial',
-        'More than 5 substantial concepts covered in the tutorial'
+        'More than 5 substantial concepts covered in the tutorial',
+        'No Substantial concepts covered',
```
Comment (espoem) [inline routes/index.js]: "No Substantial" to "No substantial"

File: views/index.ejs [hunk 8]
```diff
@@ -890,10 +890,11 @@
...
-                  					<option value="0">4-5 substantial concepts covered in the tutorial</option>
-                  					<option value="-5">2-3 substantial concepts covered in the tutorial</option>
-                  					<option value="-10">1 substantial concept covered in the tutorial</option>
-                  					<option value="-15">More than 5 substantial concepts covered in the tutorial</option>
+                  					<option value="0">4-5 substantial concepts covered in the tutorial.</option>
+                  					<option value="-5">2-3 substantial concepts covered in the tutorial.</option>
+                  					<option value="-10">1 substantial concept covered in the tutorial.</option>
+														<option value="-15">More than 5 substantial concepts covered in the tutorial.</option>
+														<option value="-30">No Substantial concepts covered.</option>
```
Comment (espoem) [inline views/index.ejs]: "No Substantial" to "No substantial"

File: views/index.ejs [hunk 9]
```diff
@@ -1747,7 +1776,7 @@
         feedend =
           category == 'translations'
             ? `).\n\n---- \nChat with us on [Discord](https://discord.gg/vMGmDSm)`
-            : `).\n\n---- \nNeed help? Write a ticket on https://support.utopian.io/. \nChat with us on [Discord](https://discord.gg/uTyJkNm). \n[[utopian-moderator]](https://join.utopian.io/)`;
+            : `).\n\n---- \n Need help? Chat with us on [Discord](https://discord.gg/uTyJkNm).\n Get in touch with Diego Pucci(elear) if you have question. \n\n[[utopian-moderator]](https://join.utopian.io/)`;
```
Comment (espoem) [inline views/index.ejs]: Remove space between \n and Need. 

Was this phrase requested? `Get in touch with Diego Pucci(elear) if you have question.` I don't think that it should be in included in comments.

**Comment by espoem** [comment]
Any reason for the big indentation of options?

File: views/index.ejs (hunk 9 above)
Comment (ms10398) [reply views/index.ejs]: I have asked Suesa once and changed it

File: views/index.ejs [hunk 10]
```diff
@@ -1776,7 +1776,7 @@
         feedend =
           category == 'translations'
             ? `).\n\n---- \nChat with us on [Discord](https://discord.gg/vMGmDSm)`
-            : `).\n\n---- \n Need help? Chat with us on [Discord](https://discord.gg/uTyJkNm).\n Get in touch with Diego Pucci(elear) if you have question. \n\n[[utopian-moderator]](https://join.utopian.io/)`;
+            : `).\n\n---- \n Need help? Chat with us on [Discord](https://discord.gg/uTyJkNm).\n\n[[utopian-moderator]](https://join.utopian.io/)`;
```
[the remark this answers is not in the document]
Comment (ms10398) [reply views/index.ejs]: Done @espoem

**Pull request merged**